clay/amphora
# Amphora authentication

### Description

* Moves all user handling logic from `amphora` into `amphora-authentication` project
* Removes test cases concerning user authentication
* Refactors routes file
* Code clean up

### QA Notes

* Do `npm install` to get `amphora-authentication` package
* Run `npm run test` to verify that all tests are passing and code coverage is at 100%
* Link the `amphora` module to a clay instance
* Add the `local` provider to the array of supported providers when setting up `amphora`
* Import a new user into your clay instance that supports local auth:
  ```yml
  -
    username: murena
    password: secret
    provider: local
    auth: admin
  ```
* Check that the user is able to login with username and password as well as with any other provider
* Verify that the `/_users` routes are working as expected (they shouldn't show the user password)

### Next Steps

* Update `amphora-authentication` module when a official release is out
